api/src/main/resources/liquibase.xml

```diff
@@ -5,14 +5,20 @@
     xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9
                   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
 
-    <changeSet id="sync2-2017-11-29" author="Soldevelo">
+    <changeSet id="sync2-2017-12-06" author="Soldevelo">
+        <preConditions onFail="MARK_RAN">
+            <not><tableExists tableName="sync_audit_message"/></not>
+        </preConditions>
         <comment>
-            Creating the sync_audit_message table
+            Creating the outgoingmessageexceptions_message table
```

Comment:
Table is named "sync_audit_message". I don't get the comment.

Reply:
You're right. I used an existing pattern for this and I didn't notice that. Thanks.